dev-python/dkimpy/dkimpy-1.0.4.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
```

Comment:
Why not 3.9? ;-)

Reply:
dev-python/pynacl and dev-python/authres do not have python3_9. But is this a good reason ?

Reply:
dev-python/pynacl and dev-python/authres (test dependencies) do not have python3_9.

Reply:
Both done now.